store/heightsub_test.go

```diff
@@ -47,6 +47,67 @@ func TestHeightSub(t *testing.T) {
 	}
 }
 
+func TestHeightSubNonAdjacement(t *testing.T) {
```

Comment:
can you fix name of this test, and also add doc to explain what it's testing for? Name is not correct

Reply:
It's correct. We're testing that `heightSub` can accept non-adj headers without a problem.